### DISCUSSION TITLE
Coordination covalent(dative) bond representation from SMILES to mol

### DISCUSSION THREAD

**Discussion by DEUTOK** [rdkit/rdkit#7844] [Development]
Chem.MolFromSmiles('[NH3][Fe]') --> conversion to smiles will give: '[NH3]->[Fe]' which is intended.
other molecules also work e.g) 'C[O-][Fe]' --> 'C[O-]->[Fe]', '[H2O][Fe]' --> '[H2O]->[Fe]'
but if you try to do this with halogen anions you get errors.

str = '[Cl-][Fe]'
mol = Chem.MolFromSmiles(str)
[16:53:41] Explicit valence for atom # 0 Cl, 2, is greater than permitted

this is the same for other halogen atoms. 

Is this some type of bugs or intended?

**Comment by greglandrum** [comment]
@DEUTOK I believe that this is a bug. Would you mind opening an issue for it?

Thanks!
